datasets/modeled/buildings/dimensions/regions.csv

```diff
@@ -0,0 +1,26 @@
+id,name
+TRE,TRE
+SPPN,SPPN
+ISNE,ISNE
+PJME,PJME
+SRSE,SRSE
+RMRG,RMRG
+CASO,CASO
+MISS,MISS
+NYCW,NYCW
+FRCC,FRCC
+NYUP,NYUP
+NWPP,NWPP
+SPPS,SPPS
+SRSG,SRSG
+MISW,MISW
+PJMW,PJMW
+PJMD,PJMD
+MISC,MISC
+SRCA,SRCA
+SPPC,SPPC
+BASN,BASN
+SRCE,SRCE
+CANO,CANO
+MISE,MISE
+PJMC,PJMC
```

Comment:
Would it be possible to give these more meaningful names (e.g., spell out the abbreviations)?

Reply:
We can — I'm not sure how much more intelligible the actual EMM Region names are, however. They are spelled out [here](https://www.eia.gov/outlooks/aeo/pdf/nerc_map.pdf) if you'd like to take a look and let me know your thoughts.

Reply:
We may as well provide more informative names. I think I'd either concatenate the NERC/ISO subregion and Geographic names, or maybe just use the Geographic name.